From e3e1603e7f5b7a03c3126d15e96d6d35a4a2e2cf Mon Sep 17 00:00:00 2001
From: reaper <theunbounddeveloper@outlook.com>
Date: Wed, 1 Jul 2026 15:37:15 -0400
Subject: [PATCH] fix(gui): seamless all-dark UI + tile custom client-list
 icons

- Unify body/.content/.banner1 to #040405 (matches REAPER.png backdrop
  ambient) and drop the middown_bg texture so the banner no longer looks
  pasted onto a lighter, non-black UI.
- Extend the red-on-black icon theme to custom-icon containers
  (.imgUserIcon/.imgUserIcon_card/.vendorIcon) and paint the View List
  popup tile (.clientlist_viewlist .client_icon > div) so custom uploaded
  client icons get the black tile + red edge like stock icons.
---
 release/src/router/www/device-map/clientlist.css | 11 ++++++++++-
 release/src/router/www/device-map/device-map.css | 11 ++++++++++-
 release/src/router/www/index_style.css           |  6 +++---
 3 files changed, 23 insertions(+), 5 deletions(-)

diff --git a/release/src/router/www/device-map/clientlist.css b/release/src/router/www/device-map/clientlist.css
index 3978025807..83178ae598 100644
--- a/release/src/router/www/device-map/clientlist.css
+++ b/release/src/router/www/device-map/clientlist.css
@@ -1191,11 +1191,20 @@ i[class^='vendor-icon'], i[class*=' vendor-icon']{
 }
 /* Client icon tiles: black background with a dark-red edge. */
 .clientIcon, .clientIcon_clicked,
-.clientIcon_no_hover, .vendorIcon_no_hover{
+.clientIcon_no_hover, .vendorIcon_no_hover,
+.vendorIcon, .imgUserIcon, .imgUserIcon_card{
 	background-color: #000000 !important;
 	box-shadow: 0px 0px 0px 1px #330000 !important;
 }
 .clientIcon:hover, .clientIcon.clicked{
 	background-color: #1A0000 !important;
 }
+/* View List popup: custom-uploaded icons sit in a transparent wrapper div
+   (only stock .clientIcon_no_hover received the black override). Paint every
+   viewlist tile black with a dark-red edge so custom icons match stock ones. */
+.clientlist_viewlist .client_icon > div{
+	background-color: #000000 !important;
+	border-color: #330000 !important;
+	box-shadow: none !important;
+}
 /* ===== END REAPER THEME ===== */
diff --git a/release/src/router/www/device-map/device-map.css b/release/src/router/www/device-map/device-map.css
index fd4eece85b..9d59fe78ad 100644
--- a/release/src/router/www/device-map/device-map.css
+++ b/release/src/router/www/device-map/device-map.css
@@ -1100,11 +1100,20 @@ i[class^='vendor-icon'], i[class*=' vendor-icon']{
 }
 /* Client icon tiles: black background with a dark-red edge. */
 .clientIcon, .clientIcon_clicked,
-.clientIcon_no_hover, .vendorIcon_no_hover{
+.clientIcon_no_hover, .vendorIcon_no_hover,
+.vendorIcon, .imgUserIcon, .imgUserIcon_card{
 	background-color: #000000 !important;
 	box-shadow: 0px 0px 0px 1px #330000 !important;
 }
 .clientIcon:hover, .clientIcon.clicked{
 	background-color: #1A0000 !important;
 }
+/* View List popup: custom-uploaded icons sit in a transparent wrapper div
+   (only stock .clientIcon_no_hover received the black override). Paint every
+   viewlist tile black with a dark-red edge so custom icons match stock ones. */
+.clientlist_viewlist .client_icon > div{
+	background-color: #000000 !important;
+	border-color: #330000 !important;
+	box-shadow: none !important;
+}
 /* ===== END REAPER THEME ===== */
diff --git a/release/src/router/www/index_style.css b/release/src/router/www/index_style.css
index 7f2c412316..375e4391ea 100644
--- a/release/src/router/www/index_style.css
+++ b/release/src/router/www/index_style.css
@@ -1,6 +1,6 @@
 /* CSS Document */
 body {
-	background-color:#0d0d0d;
+	background-color:#040405;
 	/*background: url(/images/bg_svg.png) repeat-x;*/
 	margin:10px auto;
 }
@@ -8,7 +8,7 @@ body {
 .content{
 	width: 998px;
 	border:0px;
-	background:url(images/New_ui/middown_bg.png);
+	background-color:#040405;
 	padding-bottom:24px;	/*for FAQ bottom searching bar */
 	
 	margin-top:-5px \0;	/* \0 for IE8-9 to move upper for covering the hidden form By Viz*/
@@ -40,7 +40,7 @@ body {
 .banner1{
 	width:998px;
 	height:54px;
-	background:#0b0b0b;
+	background:#040405;
 	margin:0px auto;
 }
 
-- 
2.25.1

